<ui version="4.0" >
 <class>PaletteListWidgetItemClass</class>
 <widget class="QFrame" name="PaletteListWidgetItemClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>70</height>
   </rect>
  </property>
  <property name="lineWidth" >
   <number>0</number>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="leftMargin" >
    <number>20</number>
   </property>
   <property name="topMargin" >
    <number>4</number>
   </property>
   <property name="rightMargin" >
    <number>2</number>
   </property>
   <property name="bottomMargin" >
    <number>5</number>
   </property>
   <property name="horizontalSpacing" >
    <number>2</number>
   </property>
   <property name="verticalSpacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="MoodBox::PaletteViewWidget" native="1" name="paletteWidget" >
     <property name="minimumSize" >
      <size>
       <width>163</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>163</width>
       <height>41</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QWidget" native="1" name="removeButtonHost" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>45</width>
       <height>20</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <property name="leftMargin" >
       <number>2</number>
      </property>
      <property name="topMargin" >
       <number>2</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>2</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item row="0" column="2" >
       <widget class="QToolButton" name="addPaletteButton" >
        <property name="minimumSize" >
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="accessibleName" >
         <string>transparentButton</string>
        </property>
        <property name="iconSize" >
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" >
       <widget class="QToolButton" name="deletePaletteButton" >
        <property name="minimumSize" >
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="accessibleName" >
         <string>transparentButton</string>
        </property>
        <property name="iconSize" >
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="nameLabel" >
     <property name="alignment" >
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0" />
 <customwidgets>
  <customwidget>
   <class>MoodBox::PaletteViewWidget</class>
   <extends>QWidget</extends>
   <header>paletteviewwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
